src/cli.ts

```diff
@@ -65,6 +74,16 @@ async function handler(options: Config) {
         });
       }
 
+      if (!config.outputFileFormat) {
+        questions.push({
+          type: "list",
+          name: "outputFileFormat",
+          message: messages.outputFileFormat,
+          choices: ["json", "markdown", "human_readable_markdown"],
```

Comment:
Nitpick: It occurs to me that we could store this array as a constant somewhere, so we can use it on this line and on src/config.ts.